Sources/OpenWeatherKit/Public/Forecast/HourWeather.swift

```diff
@@ -54,6 +54,11 @@ public struct HourWeather: Sendable {
     /// The pressure trend, or barometric tendency, is the kind and amount of atmospheric pressure
     /// change over time.
     public var pressureTrend: PressureTrend
+    
+    /// The rate at which snow crystals are falling, in millimeters per hour.
+    public var snowfallIntensity: Measurement<UnitSpeed>?
+    
+    public var snowfallAmount: Measurement<UnitLength>?
```

Comment:
Defining these as non-optional so as to be consistent with native framework docs, as is the goal of this project.

```suggestion
    public var snowfallIntensity: Measurement<UnitSpeed>
    
    public var snowfallAmount: Measurement<UnitLength>
```